chat-plugins/info.js

```diff
@@ -268,6 +91,9 @@ var commands = exports.commands = {
 			this.sendReply('|raw|' + hiddenrooms);
 		}
 	},
+	whoishelp:
```

Comment:
I was trying to have some visual distinction between help entries and aliases, but I understand why you want it that way. I'll change it.